ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -589,6 +589,12 @@ object Trees {
     val tpe = AnyType
   }
 
+  /** Like [[LoadModule]] but for a JS module class. */
+  case class JSLoadModule(cls: ClassType)(
+      implicit val pos: Position) extends Tree {
+    val tpe = AnyType
+  }
+
```

Comment:
Indeed. It didn't bother me that much, since "loading" a constructor is nonsensical in non-js land anyways... But maybe it is more consistent indeed.